bin/daenerys

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env php
+<?php
+function includeIfExists($file) 
+{
+    if (file_exists($file)) {
+        return include $file;
+    }
+}
+
+// Dance to find the autoloader.
+// TOOD: change this to open up the Composer config and use $c['config']['vendor-dir'] instead of "vendor"
+includeIfExists(getcwd() . '/vendor/autoload.php') ||
+includeIfExists(__DIR__ . '/../vendor/autoload.php') ||
+includeIfExists(__DIR__ . '/../autoload.php');
```

Comment:
No b/c if the package is installed from a local repo, it makes a symlink to the package, so `crate/bin/daenerys` is actually something like `path/to/core/bin/daenerys/` so `__DIR__ /../vendor/` is actually the vendor for the `path/to/core`.